packages/core/src/routes/+page.svelte

```diff
@@ -1,3 +1,25 @@
-<!-- Chirping -->
+<script lang='ts'>
```

Comment:
I added a simple Threlte "Hello World!" here since it made quick debugging easier. It shouldn't affect published library output but if it isn't desired I can remove!